### ISSUE TITLE
Simplify device with prebuilt script

### ISSUE THREAD

**Issue by ViViDboarder** [pywemo/pywemo#740]
Throwback to #161! While #179 has been merged for quite some time, I'm sure just about everyone trying to setup has, like me, dropped into ipython or something. To make setup easier for folks, I've written a script that will prompt a user to confirm devices, then for SSID and passphrase info.

I think this could be nice if it were added as a script and installed in path when installing pywemo, but I wanted to see what the maintainers think first.

The script for anyone interested in using it is here: https://gist.github.com/ViViDboarder/6ad9da8bae86baee4f11bff5ae3f1231

**Comment by Spectre5** [comment]
Hi @ViViDboarder.  When I was first creating the reset/setup code, I created a tool for that purpose [here](https://github.com/Spectre5/pywemo-setup).  I don't think it was written particularly well (which is partly why I never tried to add it to pywemo itself).  That said, it worked great for me back then, though it is probably fairly out of date with the various improvements to pyWeMo and (may?) no longer working without some updates.

In your script, how do you use it?  Assuming you reset a WeMo device, then you would need to first connect to the devices broadcasted wifi network, then run your script to discover it on its own network.  Is that right?

The repository I linked above was setup to mass automate that entire process.  So during that testing phase, I used that repo to discover all WeMo devices and reset them all.  Then that tool would look for all of the WeMo.* broadcast networks (using nmcli), connect to each individual network and setup the device found there, then move on the next.

**Comment by ViViDboarder** [comment]
Correct. The one I linked assumes you’re connecting to a single WeMo for resetting. I used it attached to that devices AP. It could be updated to allow resetting and multiple selections, but setting up new devices is a pretty common use case. Especially since reset is something one can do manually with the device itself. 

I had changed my AP password, so I had to reset my WeMo with the button and then used this script to connect it again.
